networking/networking_operators/cluster-network-operator.adoc

```diff
@@ -25,3 +25,5 @@ include::modules/nw-operator-cr.adoc[leveloffset=+1]
 == Additional resources
 * xref:../../rest_api/operator_apis/network-operator-openshift-io-v1.adoc#network-operator-openshift-io-v1[`Network` API in the `operator.openshift.io` API group]
 * xref:../../networking/configuring-cluster-network-range.adoc#nw-cluster-network-range-edit_configuring-cluster-network-range[Expanding the cluster network IP address range]
+
```

Comment:
The link is jumping to a site outside our docs server. A customer might get annoyed by this as they expect to stay on the same page or close too it. Consider adding the source destination in parentheses, as outlined in the RH supp guide:

![Screenshot from 2025-02-06 10-44-50](https://github.com/user-attachments/assets/2c20bfb0-865a-4d22-97a0-09030d729f60)

Reply:
Darragh, I accidentally merged this before I changed this. I will fix this in a followup PR. Sorry.